INN/umbrella-caribbean
# fix some translated strings and add Grunt toolchain for translation files

## Changes

This pull request makes the following changes:

- adds text domains for a couple of `__(` calls
- adds Largo's latest trunk: https://github.com/INN/largo/compare/INN:v0.7-prerelease1...590181982d22a5444eb3c5ccca58ea8b56db12f7
    - changes an ID `#header-social` to a class name `header-social` in https://github.com/INN/largo/pull/1826 — doesn't affect any Caribbean styles
    - XAMPP fixes: https://github.com/INN/largo/pull/1850
    - the most-recent round of translatability fixes: https://github.com/INN/largo/pull/1858

## Why



For https://github.com/INN/umbrella-caribbean/issues/96

## Testing/Questions

Features that this PR affects:

- labels on a couple blocks
- `grunt`
- things that were already tested in Largo


Questions that need to be answered before merging:

- [ ] what other testing is needed?

Steps to test the grunt toolkits:

1. `npm i`
2. `grunt`

Steps to test the translation stuff:

1. Install https://wordpress.org/plugins/theme-check/ and run it against the `caribbean` theme
2. The Theme Check results should not come back with any errors regarding translation.
